pages/linux/e2fsck.md

```diff
@@ -1,16 +1,20 @@
 # e2fsck
 
-> Check a Linux ext2/ext3/ext4 filesystem. The filesystem should be unmounted at the time the command is run.
+> Check a Linux ext2/ext3/ext4 filesystem. Unmount the partition before running a command.
```

Comment:
I prefer the prior version of this line, its a little more descriptive, and at a first glance, I thought this command would unmount it automatically, it seems like a statement rather than a command.

Reply:
I think the original sentence is too long. What about adding an exclamation mark at the end?